commerce/domain/orchestrators/classes/CartCalculateExecutorForUnitTest.cls

```diff
@@ -0,0 +1,25 @@
+/**
+* @description Sample mock executor for unit testing a custom orchestrator within Cart Calculate API.
+*/
+global class CartCalculateExecutorForUnitTest extends CartExtension.CartCalculateExecutorMock {
+
+    /**
+     * @description All classes extending CartExtension.CartCalculateExecutorMock must have a default constructor defined
+     */
+    global CartCalculateExecutorForUnitTest() {}
+
+    global override void prices(CartExtension.CartCalculateCalculatorRequest request) {
+        CartExtension.Cart cart = request.getCart();
+        cart.setName('CartRepriced');
+    }
+
+    global override void promotions(CartExtension.CartCalculateCalculatorRequest request) {}
```

Comment:
why don't we implement all this methods in the Sample?
`cart.setName(cart.getName() + 'PromotionsRecalculated');`